libsql-server/src/http/admin/mod.rs

```diff
@@ -242,7 +243,10 @@ async fn handle_create_namespace<M: MakeNamespace, C: Connector>(
     };
 
     let namespace = NamespaceName::from_string(namespace)?;
-    app_state.namespaces.create(namespace.clone(), dump).await?;
+    app_state
+        .namespaces
+        .create(namespace.clone(), dump, req.bottomless_db_id)
```

Comment:
I think we should make this fn accept a struct instead of parameters. This is going to conflict with the auth work I was doing. If possible I'd like to maybe hold off on merging this now and then I can merge my code then rebase this one ontop of it? Threading through stuff through this interface is a PITA so I can make that struct change then we can rebase this ontop to make things easier.

Reply:
I've opened a draft PR with my latest work rebased, https://github.com/tursodatabase/libsql/pull/616 lets chat about this tomorrow.